src/main/java/org/fundacionjala/core/utils/JsonFilesReader.java

```
@@ -0,0 +1,36 @@
+package org.fundacionjala.core.utils;
+
+import org.json.simple.JSONArray;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public final class JsonFilesReader {
+
+    private JsonFilesReader() {
+    }
+
+    /**
+     * Try to read a json file and return a JSONArray with the read info.
+     * @param filePath
+     * @return JSONArray;
+     */
+    public static JSONArray jsonArrayFromJsonFile(final String filePath) {
+        try {
+            JSONParser jsonParser = new JSONParser();
+            FileReader reader = new FileReader(filePath);
+            Object obj = jsonParser.parse(reader);
+            return (JSONArray) obj;
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Avoid having the catch block if you are just printing the stacktrace because you are hiding the error, the test will keep running and you will have a failure in a different method that needed the infromation from the json file.